dashboards/component/stat/serializers.py

```diff
@@ -41,6 +42,7 @@ class Meta(ClassWithMeta.Meta):
         model: Optional[Model] = None
         title: Optional[str] = ""
         unit: Optional[str] = ""
+        filter_component: Optional[Type[FilterComponent]] = None
```

Comment:
don't think this needs to be here - should be passed into serialize instead as an attr